Jenkinsfile.migration

```diff
@@ -27,12 +27,16 @@ pipeline {
 				dropdb lisk_test || true
 				createdb lisk_test
 				cd "${params.MIGRATE_FROM}"
-				cp test/config.json .
-				cp test/genesis_block.json .
+				cd test
```

Comment:
Why not `cp genesis*lock.json .` instead of changing directory, using find and then changing directory again?
(possibly even better: `cp genesis{_b,B}lock.json .`)